circle.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/cactus
+    docker:
+      - image: circleci/openjdk:8-jdk-browsers
+    steps:
+      - checkout
+
+      - run:
+          name: Install Dependencies
+          command: ./gradlew dependencies
+
+      - run:
+          name: Test
+          command: ./gradlew check
+
+      - deploy:
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
Circleci 2.0 is still getting mature. I know this way sounds wrong, but it's getting there.